yoti_python_sdk/profile.py

```diff
@@ -161,6 +161,27 @@ def document_images(self):
     def document_details(self):
         return self.get_attribute(config.ATTRIBUTE_DOCUMENT_DETAILS)
 
+    def get_age_verifications(self):
```

Comment:
Can we add some test coverage for these new methods?

Reply:
Added them [here](https://github.com/getyoti/yoti-python-sdk/pull/98/commits/336489b9d561540e0afacb95c40b42224b461bac)